### ISSUE TITLE
[Question] DotIndicator always on

### ISSUE THREAD

**Issue by GabriPalmyro** [imaNNeo/fl_chart#1771]
**Is your feature request relasted to a problem? Please describe.**
Hi guys, I have two questions:
- Is it possible to create the chart with the indicator dot always selected (even if its not pressed)?
- Is it possible to create a gradient in the graph line and change it with the user interaction?

**Describe alternatives you've considered**

**Additional context**
<img width="702" alt="Captura de Tela 2024-11-11 aÌs 17 14 45" src="https://github.com/user-attachments/assets/dcb43399-8b94-42e4-b377-930ddf4cbfd3">

**Comment by Sosssen** [comment]
yes, both options are available, please check sample line charts (from example app or here https://app.flchart.dev/#/line)

* 'indicator while not pressed' is visible on line chart sample 5
* gradient is visible on line chart sample 3 (you can change it depending on user actions)

there are also source codes attached to each chart:)
